rpm_spec/subpackages/manageiq-core

```diff
@@ -35,8 +35,8 @@ done
 # These directories contain files not owned by this rpm.
 #  For upgrades, ensure the files have the correct group privs
 #  so root and manageiq users can read them.
-[ -e %{app_root}/certs/v2_key ] && %{__chown} manageiq.manageiq %{app_root}/certs/v2_key
-[ -e %{app_root}/certs/v2_key ] && %{__chmod} o-rw %{app_root}/certs/v2_key
+[ -e %{app_root}/certs/v2_key ] && %{__chown} manageiq.manageiq %{app_root}/certs/*
+[ -e %{app_root}/certs/v2_key ] && %{__chmod} o-rw %{app_root}/certs/*
```

Comment:
Will this cause apache to not be able to read the server.cer?

Reply:
I'm pretty sure Apache is more concerned with the selinux privileges.
It should be running as `root` at this time so should be able to read it.

But yea, that is why this is WIP.

Reply:
Also, when the selinux privileges are off, then it is confusing because it is running as `root` and the error messages look like a `chmod` issue. `ls -Z certs` and `restorecon -R certs` is our friend when this happens.